test/helpers/PlayerInteractionWrapper.js

```diff
@@ -309,18 +309,15 @@ class PlayerInteractionWrapper {
      * Sets the contents of the conflict discard pile
      * @param {String[]} newContents - list of names of cards to be put in conflict discard
      */
-    setDiscard(newContents = []) {
+    setDiscard(newContents = [], prevLocations = ['deck']) {
```

Comment:
What happened that this needed to be changed?

Reply:
it's for bounty hunter crew, when addind card on discard the builder does found it in the deck and crash, I just copy setHand